doc/schema/schema.md

```diff
@@ -40,10 +41,20 @@ Currently there are 2 beta implementations of this design:
  <li><a class="reference internal" href="#examples">Examples</a></li>
 </ol>
 
+## Terminology
+The terminology used in this specification is based on [xarray's terminology](http://xarray.pydata.org/en/stable/terminology.html), however, no xarray knowledge is assumed in this description. There are also some extensions particular to  the InferenceData case.
+
+* **Variable**: NetCDF-like variables are multidimensional labeled arrays representing a single quantity. Variables and their dimensions must be named. They can also have attributes describing it. Relevant terms related to InferenceData variables are: *variable_name*, *values* (its data), *dimensions*, *coordinates*, and *attributes*.
+* **Dimension**: The dimensions of an object are its named axes. A variable containing 3D data can have dimensions `[chain, draw, dim0]`, thus, its `0th`-dimension is `chain`, its `1st`-dimension is `draw` and so on. Every dimension present in an InferenceData variable must share names with a *coordinate*. Given that dimensions must be named, dimension and dimension name are used equivalents.
+* **Coordinate**: A named array that labels a dimension. A coordinate named `chain` with values `[0, 1, 2, 3]` would label the `chain` dimension. Coordinate names and values can be loosely though of as labels and tick labels along a dimension. 
+* **Group**: Dataset containing one or several variables with a conceptual link between them. Variables inside a group will generally share some dimensions too.
```

Comment:
**Group**: In Arviz's `InferenceData` objects, a group is an `xarray.Dataset` containing...
For example, the `posterior` group contains a representation of the posterior distribution conditioned on the observatinos in the `observed_data` group.